frontend/src/metabase/visualizations/visualizations/Funnel.jsx

```diff
@@ -40,7 +40,7 @@ export default class Funnel extends Component {
     height: 4,
   };
 
-  static isSensible(cols, rows) {
+  static isSensible({ cols, rows }) {
```

Comment:
This change was unrelated, but I realized that it was broken and always showing the funnel option grayed out.

Reply:
I think that change might fix the confusion in #4890